pkgs/tools/security/enum4linux/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, makeWrapper, samba, perl, openldap }:
+
+stdenv.mkDerivation rec {
+  pname = "enum4linux";
+  version = "0.8.9";
+  src = fetchurl {
+    url = "https://labs.portcullis.co.uk/download/enum4linux-${version}.tar.gz";
+    sha256 = "41334df0cb1ba82db9e3212981340372bb355a8160073331d2a1610908a62d85";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [ makeWrapper samba perl openldap ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp enum4linux.pl $out/bin/
+
+    wrapProgram $out/bin/enum4linux.pl \
+      --prefix PATH : ${stdenv.lib.makeBinPath [ samba perl openldap ]}
```

Comment:
You don't need to include `perl` in `makeBinPath` because nix will automatically modify the `#!/usr/bin/perl` shebang at the top of the file for you (because you listed `perl` in your `buildInputs`) to a hard coded path in the nix store.